src/components/merchantPage/MenuInfo.jsx

```diff
@@ -32,11 +28,10 @@ const MenuInfo = ({
                     />
                 </button>
             </div>
-            <h2 className="text-2xl font-bold text-black">{storeName}</h2>
-
+            <h2 className="text-2xl font-bold text-black">{name}</h2>
             <div className="flex items-center mt-1">
                 <div className="font-medium text-[13px] leading-[15px] text-gray-600 mb-[-1px]">
-                    {starRate}
+                    {rating}
                 </div>
                 <div className="text-yellow-500">
                     {[...Array(solidStar)].map((_, i) => (
```

Comment:
The stars are not displaying the corresponding star styles according to the rating number. If possible, it would be best to show the effect of half stars.